src/components/editor/Store/create.ts

```diff
@@ -147,6 +217,9 @@ const getInitialState = <T>(
 
 export const createEditorStore = <T>(key: string) => {
     return create<EditorStoreState<T>>()(
-        devtools((set) => getInitialState<T>(set), devtoolsOptions(key))
+        devtools(
+            (set, get) => getInitialState<T>(set, get),
```

Comment:
Thinking out loud - starting to think we should just always send `set` and `get` to all stores when starting 'em up and just mark them as unused.

Reply:
Agreed. That has been my position.